userspace/falco/app/actions/init_falco_engine.cpp

```diff
@@ -117,7 +117,7 @@ falco::app::run_result falco::app::actions::init_falco_engine(falco::app::state&
 		auto manager = s.offline_inspector->get_plugin_manager();
 		for (const auto &p : manager->plugins())
 		{
-			if (p->caps() & CAP_SOURCING)
+			if (p->caps() & CAP_SOURCING && p->id() != 0)
```

Comment:
this check is used in more than one place probably at a certain point we could create an helper for it

Reply:
I agree, i think a small method would help making the code more readable.